app/services/finance/stripe_charge_service.rb

```diff
@@ -3,12 +3,17 @@
 class Finance::StripeChargeService
   PAYMENT_INTENT_SUCCEEDED = 'succeeded'
   PAYMENT_INTENT_REQUIRES_CONFIRMATION = 'requires_confirmation'
+  PAYMENT_DESCRIPTION = 'API services'
 
   def initialize(gateway, payment_method_id:, invoice: nil, gateway_options: {})
     @gateway = gateway
     @payment_method_id = payment_method_id
     @invoice = invoice
     @gateway_options = gateway_options
+
+    # As per Indian regulations the Payment intents should have a description,
+    # see https://stripe.com/docs/india-accept-international-payments#valid-charges
+    @gateway_options[:description] = charge_description
```

Comment:
I don't think we should hardcode this option. Maybe:
```suggestion
    @gateway_options[:description] ||= charge_description
```